validator/example/cmd_test.go

```diff
@@ -0,0 +1,39 @@
+package example
+
+import (
+	"bytes"
+	"io/ioutil"
+	"testing"
+
+	"github.com/aerogear/charmil/validator/rules"
+)
+
+func Test_ExecuteCommand(t *testing.T) {
+	cmd := NewCommand()
+
+	// Testing cobra commands with default recommended config
+	// default config can also be overrided by
+	/*
+		var vali rules.RuleConfig = rules.RuleConfig{
+				Verbose: true,
+				MustExist: rules.MustExist{
+					Fields: []string{"Args"},
+				},
+			}
+	*/
+	var vali rules.RuleConfig
+	validationErr := vali.ExecuteRules(cmd)
+	for _, errs := range validationErr {
```

Comment:
This is not test :) 
Test does asertions :)
We should at least check if no errors are returned.
We can also print out some info etc.

THis test should be passing as we going to use it for CI/CD #93